packages/core/src/browser/widget-manager.spec.ts

```diff
@@ -3,9 +3,13 @@
  *
  * Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except in compliance with the License.
  * You may obtain a copy of the License at http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * @jest-environment @theia/core/src/browser/test/jsdom-environment
```

Comment:
`widget-manager.spec.ts` did not need jsdom before, right? is it needed now?

Reply:
At least it requires `navigator` somewhere and failed the test.

Reply:
Verified with `master`. Thanks!